tools/swift-inspect/Sources/swift-inspect/String+Extensions.swift

```diff
@@ -11,10 +11,32 @@
 //===----------------------------------------------------------------------===//
 
 import SwiftRemoteMirror
+import Foundation
 
 extension DefaultStringInterpolation {
   mutating func appendInterpolation<T>(hex: T) where T: BinaryInteger {
     appendInterpolation("0x")
     appendInterpolation(String(hex, radix: 16))
   }
 }
+
+enum Std {
+  struct Err: TextOutputStream {
```

Comment:
I'd like to keep this as a more general-purpose `File` type, which is then initialized with stderr for the value of `static var err` below. It will make it nice to use this for stdout or other files if we ever need to.